puppet.te

```diff
@@ -350,6 +350,7 @@ optional_policy(`
 ')
 
 optional_policy(`
+	systemd_dbus_chat_timedated(puppetagent_t)
 	systemd_dbus_chat_timedated(puppetmaster_t)
```

Comment:
access for puppetmaster_t was already set. but I don't know what business puppetmater_t has with timedated. Maybe sliped in errorrnous?